test/test_python_api.py

```
@@ -0,0 +1,21 @@
+cif_file = pathlib.Path(__file__).parent / "1cbs.cif.gz"
+json_file = pathlib.Path(__file__).parent / "1cbs.json"
+
+def test_compute_stats():
+    stats = py_tortoize.tortoize_compute_stats(str(cif_file))
+    with open(json_file) as fp:
+        ref_stats = json.load(fp)
+
+    for n in range(len(stats["model"]["1"]["residues"])):
+        ress, resr = stats["model"]["1"]["residues"][n], ref_stats["model"]["1"]["residues"][n]
+        if any([ress[k] != resr[k] for k in ("asymID", "compID", "seqID", "pdb")]):
+            raise ValueError(f"Failed basic residue data at {n}: DO NOT rely on this installation of py_tortoize")
```

Ruff (0.14.14)

[warning] Avoid specifying long messages outside the exception class

(TRY003)